### DISCUSSION TITLE
docker-compose --env "./myconfig.env" up - Command does not complete

### DISCUSSION THREAD

**Discussion by sodgaard** [tillsteinbach/VWsFriend#214] [Q&A]
Hi,

After updating the docker-compose.yml and running "docker-compose pull" and subsequently the above command it seems not to complete and launch VWsFriend in the background. Looking at containers VWsFriend no longer shows up. The first "docker_" are related to TeslaMate which is functioning. The "VWsFriend_" are - well obviously - related to the VWsFriend. :-)
![image](https://user-images.githubusercontent.com/44952678/162280814-adad0082-9543-4234-a2c0-da1ec718b2c7.png)

The SSH window shows:
![image](https://user-images.githubusercontent.com/44952678/162281415-f9f85862-a0a0-484e-898b-6d225e66c9f9.png)

It seems like a new data request is sent every 5 mints triggering a new logline. However the last line does not look good.
This should just happen in the background, right?

**Comment by sodgaard** [comment]
Tried running the docker-compose file with the "-d" at the end.
A subsequent "docker ps" gives the stats (where VWsFriend is indicated as unhealthy). Also it still does not show up in the NAS Docker GUI.:
![image](https://user-images.githubusercontent.com/44952678/162284573-7af848eb-d0f6-4a66-9249-fd74a2a720c9.png)

**Comment by sodgaard** [comment]
Till - could it be because the Healthcheck still explicitly refers to port 4000?

```
healthcheck:
      test: (wget -qO- http://localhost:4000/healthcheck | grep "ok" -q) || exit 1
      interval: 60s
      timeout: 10s
      retries: 3
```

/Stephen

**Reply by sodgaard** [reply]
Manually changing the port on the "test" line to the one choosen in the .env file - then it is considered healthy and shows up as a running container...

**Reply by tillsteinbach** [reply]
Yes! Thanks for the hint! I need to change that line!

**Reply by tillsteinbach** [reply]
Done
